index.html

```diff
@@ -0,0 +1,1048 @@
+            </a>
+          </li>
+          <li>
+            <a class="products__link" href="./catalog.html">
+              <article class="products__list-item list list__column">
+                <div class="list-item__image">
+                  <svg class="list-item__icon-image">
+                    <use
+                      xlink:href="./images/icons/sprite.svg#presentation"
+                    ></use>
+                  </svg>
+                </div>
+                <h3 class="products__title heading__article-title">
+                  Presentation
+                </h3>
+              </article>
+            </a>
+          </li>
+          <li>
+            <a class="products__link" href="./catalog.html">
+              <article class="products__list-item list list__column">
+                <div class="list-item__image">
+                  <svg class="list-item__icon-image">
+                    <use xlink:href="./images/icons/sprite.svg#font"></use>
+                  </svg>
+                </div>
+                <h3 class="products__title heading__article-title">Fonts</h3>
+              </article>
+            </a>
+          </li>
+          <li>
+            <a class="products__link" href="./catalog.html">
+              <article class="products__list-item list list__column">
+                <div class="list-item__image">
+                  <svg class="list-item__icon-image">
+                    <use
+                      xlink:href="./images/icons/sprite.svg#illustration"
+                    ></use>
+                  </svg>
+                </div>
+                <h3 class="products__title heading__article-title">
+                  Illustrations
+                </h3>
+              </article>
+            </a>
+          </li>
+        </ul>
+      </section>
+      <section class="categories container__section list">
+        <div class="categories__categories-information list list__column">
+          <h2 class="categories-information__title heading__section">
+            categories
+          </h2>
+          <strong
+            class="categories-information__highlite-text heading__section-title"
+          >
+            Discover Assets for Every Projects
+          </strong>
+          <p class="categories-information__description text__section">
+            From fonts and icons to mockups and templates, our comprehensive
+            categories ensure that you find exactly what you need to your
+            projects.
+          </p>
+          <a
+            href="./catalog.html"
+            class="categories-information__button button button__primary button__large"
+          >
+            Shop Now
+          </a>
+        </div>
+        <div class="categories__list list">
+          <ul
+            class="categories__categories-cards list list__column list__elements"
+          >
+            <li>
+              <a href="./catalog.html" class="categories-cards__link">
+                <article
```

Comment:
The article element represents a complete, or self-contained, composition in a document.

I think we can't use it as a self-contained composition, so you need to remove this tag.

Reply:
Fixed 🥇